src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -1070,13 +1070,19 @@ public function testDumpFile()
     {
         $filename = $this->workspace.DIRECTORY_SEPARATOR.'foo'.DIRECTORY_SEPARATOR.'baz.txt';
 
+        // skip mode check on Windows
+        if ('\\' !== DIRECTORY_SEPARATOR) {
+            $oldMask = umask(0002);
```

Comment:
I agree, removed the extra tests, because I will create a separate pull request for enhancing the test and taking the umask into account for 2.3.